portfolio/src/main/webapp/style.css

```diff
@@ -1,10 +1,153 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body,
+html {
+    height: 100%;
+    width: 100%
 }
+ 
+body,
```

Comment:
I noticed that body#font-weight is overridden in the next block - should this be delete?